compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/MultiLayerComposeScene.skiko.kt

```diff
@@ -579,7 +581,10 @@ private class MultiLayerComposeSceneImpl(
         override fun setContent(content: @Composable () -> Unit) {
             check(!isClosed) { "AttachedComposeSceneLayer is closed" }
             composition?.dispose()
-            composition = owner.setContent(parent = compositionContext) {
+            composition = owner.setContent(
+                parent = compositionContext,
+                { compositionLocalContext }
```

Comment:
```suggestion
                parent = this@AttachedComposeSceneLayer.compositionContext,
                { this@AttachedComposeSceneLayer.compositionLocalContext }
```
To avoid dangerous situation when we remove/rename inner fields, and don't notice that we started to use the parent parameters.